cmd/vic-machine/create/create.go

```diff
@@ -1292,61 +1312,80 @@ func logArguments(cliContext *cli.Context) {
 				continue
 			}
 			url.User = nil
-			log.Debugf("--target=%s", url.String())
+			flag := fmt.Sprintf("--target=%s", url.String())
+			log.Debug(flag)
+			args = append(args, flag)
```

Comment:
I'm not sure what you mean as only flags that were explicitly specified by the user will make it into this path, so defaults should not be persisted. A user can supply a default explicitly, and that would be persisted, for example:  `--no-tls-verify=false`. Can you please elaborate in case I'm misunderstanding?